build.sbt

```diff
@@ -915,7 +915,6 @@ lazy val docs = http4sProject("site")
       cryptobits,
     ),
     description := "Documentation for http4s",
-    mdocIn := (Compile / sourceDirectory).value / "mdoc",
```

Comment:
Moved the site to the `docs/` directory, which is the mdoc default and how every other project is set up.